solutions/day-06/level2/src/components/NumberCell.js

```diff
@@ -0,0 +1,26 @@
+const isPrime = (number) => {
+  for (let i = 2, s = Math.sqrt(number); i <= s; i++) {
+    if (number % i === 0) {
+      return false;
+    }
+  }
+  return number > 1;
+};
+
+const NumberCell = ({ number }) => {
+  let className = null;
+  if (isPrime(number)) {
+    className = "cell-prime";
+  } else if (number % 2 === 0) {
+    className = "cell-even";
+  } else {
+    className = "cell-odd";
+  }
```

Comment:
You can instead make the "cell-odd" the default value of the `className` variable, then remove the `else {}` block